src/tests/test_process.py

```diff
@@ -110,7 +172,16 @@ def test_shutdown(self):
     @classmethod
     def tearDownClass(cls):
         """Special teardown because pybitmessage is already stopped"""
-        cls._cleanup_files()
+        try:
+            if cls.process.is_running():
+                cls.process.kill()
```

Comment:
This looks like a cheat. `TestProcessProto` does the same: https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/tests/test_process.py#L73

Reply:
1. it isn't the same, because `TestProcessProto` first calls `_stop_process` to send a `SIGTERM`. This expects the program to already have closed on its own, and just send `SIGKILL` in an emergency.
2. it allows you to put custom handlers into `TimeoutExpired` and `NoSuchProcess` if you want the tests to be more strict. `TimeoutExpired` means that there is something very wrong as it's blocking a `SIGKILL`.
3. However, the main point here is line 178 as previously, it wasn't waiting for the program to finish after `SIGKILL`, which I think lead unpredictably to problems.

Reply:
OK, I get it:
- it doesn't disable the test because it's failing by it's own if process not stops.
- it is not the same as super